src/main/groovy/net/researchgate/release/ReleasePlugin.groovy

```diff
@@ -104,7 +104,6 @@ class ReleasePlugin extends PluginHelper implements Plugin<Project> {
             startParameter.projectProperties.putAll(project.getGradle().startParameter.projectProperties)
             startParameter.projectProperties.put('release.releasing', "true")
             startParameter.projectDir = project.projectDir
-            startParameter.settingsFile = project.getGradle().startParameter.settingsFile
```

Comment:
Please wrap this in an if that checks the gradle version to not break existing usages of it.

Reply:
@Hillkorn updated

Reply:
Thanks